gradle.properties

```diff
@@ -19,4 +19,5 @@
 #
 #
 
-protobufPluginVersion=0.8.15
\ No newline at end of file
+protobufPluginVersion=0.8.15
+spotbugsPlugin=4.7.0
```

Comment:
is this the same version that we have in Maven pom.xml ?
why the new spotbugs found issues that were not already tracked?

can you please update the pom.xml as well ?

Reply:
Yeah. That's version of spotbugs gradle plugin itself. This is version for the tool version such as engine/algorithm to find bugs which is  toolVersion = '3.1.8'.    <spotbugs-annotations.version>3.1.8</spotbugs-annotations.version>

Reply:
Hi @eolivelli - could you help review this again? Thanks